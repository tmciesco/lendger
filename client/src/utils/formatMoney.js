// For now this doesn't support change

export default function(amount) {
	const options = {
		style: "currency",
		currency: "USD",
		minimumFractionDigits: 0,
	}
	//if (amount % 100 === 0) options.minimumFractionDigits = 0
	const formatter = new Intl.NumberFormat("en-US", options)
	return formatter.format(amount / 1)
}
