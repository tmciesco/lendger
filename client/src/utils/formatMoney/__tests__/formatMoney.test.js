import formatMoney from '../../formatMoney'


it ('formats a number into currency', () => {
    let number = 10000
    expect(formatMoney(number)).toMatchSnapshot()
})