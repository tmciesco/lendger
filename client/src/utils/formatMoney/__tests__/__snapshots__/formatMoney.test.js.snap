// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formats a number into currency 1`] = `"$10,000"`;
